src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -394,21 +394,20 @@ public boolean isDefaultSystemFieldsExpanded()
      */
     public String getMode()
     {
-        return elementCache().customFieldsViewToggle.getSelectedStatus(); // will be either "Summary mode" or "Detail mode"
+        return elementCache().customFieldsViewToggle.getSelectedStatus(); // will be either "Summary" or "Detail"
```

Comment:
I know this isn't open for review yet but I have some feedback since you're refactoring these anyway. Tests shouldn't have to know the specific toggle text for getting or setting the mode.
Outside of this class, the mode could be represented by an `enum` (e.g. `public enum DomainFormMode { Summary, Detail }`) which the public setter and getter would use.
Another option would be to get rid of (or make private) the existing `getMode`/`switchMode` methods and restrict callers to using `isDetailMode`/`isSummaryMode`/`selectDetailMode`/`selectSummaryMode`.

Reply:
Good suggestion. I'll look into making those changes and ping you for review once ready.

Reply:
Leaving this PR review for @labkey-tchad.

Reply:
Done